tests/lib/rules/jsx-curly-spacing.js

```diff
@@ -756,14 +756,6 @@ ruleTester.run('jsx-curly-spacing', rule, {
       `,
       options: ['always'],
     },
-    {
-      code: `
-        <App {
-        ...bar
-        } />;
-      `,
-      options: ['always'],
-    },
```

Comment:
Duplicate of:

https://github.com/jsx-eslint/eslint-plugin-react/blob/8e1a94b67d081fdc132e9a7e175db3fbf2e02956/tests/lib/rules/jsx-curly-spacing.js#L751-L758